search-app/src/cmr/search/data/elastic_search_index.clj

```diff
@@ -36,10 +36,18 @@
   (let [fetched-index-set (index-set/get-index-set context index-set-id)
         rebalancing-collections (get-in fetched-index-set
                                         [:index-set :granule :rebalancing-collections])
+        ;; We want to make sure anything in the rebalancing collections index that has a
+        ;; target of a separate granule index continues to use the small collections index for now.
+        rebalancing-targets-map (get-in fetched-index-set [:index-set :granule :rebalancing-targets])
+        moving-to-separate-index (keep (fn [[k v]]
```

Comment:
I probably would have put this anonymous function in a separate function; at this level of simplicity I'd say it's just personal taste.

Reply:
Done - also helped me realize other code in this function wasn't even needed.